src/docs/styles/docs-global.scss

```diff
@@ -63,6 +64,10 @@ p code {
   font-size: inherit;
 }
 
+p strong {
+  color: #ffffff;
```

Comment:
Will the docs support theming in the future?

Reply:
good point, I updated it with the contrasting color of the background:

```
p strong {
  color: var(--background-contrast);
}
```